treeherder/webapp/api/performance_data.py

```diff
@@ -1079,6 +1083,19 @@ def list(self, request):
 
         return Response(data=serialized_data)
 
+    def _get_signature_based_properties(self, sig, option_collection_map):
+        suite = sig.get("suite", "")
+        test = sig.get("test", "")
+        lower_is_better = sig.get("lower_is_better", "")
+        sig_hash = sig.get("signature_hash", "")
+        option_name = self._get_option_name(sig, option_collection_map)
+        extra_options = sig.get("extra_options", "")
+        return extra_options, lower_is_better, option_name, sig_hash, suite, test
```

Comment:
nit: this could be something like the following since you don't need any of variables that you defined:
```
return (
    sig.get("suite", ""),
    ...
)
```